### ISSUE TITLE
Errors when loading diagram

### ISSUE THREAD

**Issue by Alexander-Wilms** [gaphor/gaphor#89]
### I'm submitting a...
- [x] Bug report  
- [ ] Feature request
- [ ] Documentation issue or request

### Current behavior
I tried loading test-diagrams/diagram-#4.gaphor and the diagram only contained the interface "Room". On the terminal, there were a lot of error messages:

```
flatpak run --filesystem=home org.gaphor.Gaphor 
Application INFO initializing service service.component_registry
Application INFO initializing service service.main_window
gaphor.ui.stock INFO Icon dir: /app/lib/python3.7/site-packages/gaphor/ui/pixmaps
Application INFO initializing service service.action_manager
ActionManager INFO Loading action provider services
Application INFO initializing service service.ui_manager
Application INFO initializing service service.properties
Application INFO initializing service service.adapter_loader
Application INFO initializing service service.alignment
Application INFO initializing service service.copy
Application INFO initializing service service.diagram_export_manager
Application INFO initializing service service.diagram_layout
Application INFO initializing service service.element_dispatcher
Application INFO initializing service service.element_factory
Application INFO initializing service service.file_manager
gaphor.services.filemanager INFO Updating recent files
Application INFO initializing service service.help
HelpService INFO Starting
Application INFO initializing service service.pynsource
Application INFO initializing service service.sanitizer
Application INFO initializing service service.undo_manager
UndoManager INFO Starting
Application INFO initializing service service.xmi_export
gaphor.services.filemanager INFO Setting current file
gaphor.services.filemanager INFO Loading file
gaphor.storage.storage INFO Loading file b'diagram-#4.gaphor'
b'/home/awilms/Development/gaphor/test-diagrams/diagram-#4.gaphor'
gaphor.storage.storage INFO Read 149 elements from file
version 0.14.1
gaphor.ui.diagramtools ERROR Error during connect
Traceback (most recent call last):
  File "/app/lib/python3.7/site-packages/gaphor/ui/diagramtools.py", line 92, in connect
    adapter.connect(handle, sink.port)
  File "/app/lib/python3.7/site-packages/gaphor/adapters/connectors.py", line 429, in connect
    self.connect_connected_items()
  File "/app/lib/python3.7/site-packages/gaphor/adapters/connectors.py", line 373, in connect_connected_items
    assert adapter
AssertionError
Traceback (most recent call last):
  File "/app/lib/python3.7/site-packages/gaphas/view.py", line 867, in on_draw
    Context(cairo=cr, items=self.get_items_in_rectangle(area), area=area)
  File "/app/lib/python3.7/site-packages/gaphas/painter.py", line 83, in paint
    painter.paint(context)
  File "/app/lib/python3.7/site-packages/gaphas/painter.py", line 155, in paint
    self._draw_items(context.items, cairo, context.area)
  File "/app/lib/python3.7/site-packages/gaphas/painter.py", line 132, in _draw_items
    self._draw_item(item, cairo, area=area)
  File "/app/lib/python3.7/site-packages/gaphas/painter.py", line 120, in _draw_item
    draw_all=self.draw_all,
  File "/app/lib/python3.7/site-packages/gaphor/diagram/compartment.py", line 541, in draw_compartment
    comp.draw(context)
  File "/app/lib/python3.7/site-packages/gaphor/diagram/compartment.py", line 189, in draw
    item.draw(context)
  File "/app/lib/python3.7/site-packages/gaphor/diagram/compartment.py", line 85, in draw
```

The`TypeError: Pango.Layout.set_text() takes exactly 3 arguments (2 given)` repeats quite often.

### Expected behavior
There should be no errors.

### Minimal reproduction of the problem with instructions
Open test-diagrams/diagram-#4.gaphor in the current Flatpak version of Gaphor (1.0.0rc1)

### System Information
OS:
- [x] Linux (Please put in notes the specific distro)
- [ ] MacOS
- [ ] Windows 10
- [ ] Windows 7

NOTES:
Distro: Fedora 29 KDE

**Comment by amolenaar** [comment]
I ran into this issue before and it has been fixed in 279280a1a.

**Comment by Alexander-Wilms** [comment]
Ah, great
